.gitignore

```diff
@@ -22,3 +22,8 @@ data/seed.yaml.generated
 tests/package-lock.json
 my.db
 mount/conformance/
+
+node_modules/*
+package-lock.json
+package.json
+
```

Comment:
we don't need those in root as `make run-tests` is changing the working directory to `tests`

Reply:
Those are just .gitignore additions